wiki-webui/src/main/java/org/exoplatform/wiki/commons/Utils.java

```diff
@@ -255,7 +254,6 @@ public static void feedDataForWYSIWYGEditor(UIWikiPageEditForm pageEditForm, Str
       markup = (markupInput.getValue() == null) ? "" : markupInput.getValue();
     }
     String xhtmlContent = renderingService.render(markup, markupSyntax, Syntax.ANNOTATED_XHTML_1_0.toIdString(), false);
-    xhtmlContent = StringCommonUtils.encodeScriptMarkup(xhtmlContent);

```

Comment:
Ok will try other approach, but please note that reasons for reverting are listed in the jira issue and the fix was made to fix the maintenance branch 4.3.x as a deeply fix will be made for 4.4 and can't be backported for the moment as new  
sanitizer lib will be used.